### ISSUE TITLE
Duplicate menu items

### ISSUE THREAD

**Issue by Gangwolf** [nylas/N1#418]
N1 Version 0.3.23
Windows 10

While changing some preferences I saw a menu item referring to the current application window. Also some menu items duplicate functionality within the same app window, basically redundant menu items like the 'Add Account' item in the preferences window.

**Comment by mbilker** [comment]
Picture?

**Comment by Gangwolf** [comment]
![n1 duplicate menu items 418](https://cloud.githubusercontent.com/assets/15684336/11229472/5fef8198-8d4a-11e5-93d8-7a8978beeb3d.PNG)

**Comment by Gangwolf** [comment]
![n1 duplicate menu items 418](https://cloud.githubusercontent.com/assets/15684336/11229484/79d13a3e-8d4a-11e5-9322-f5f68f185d98.PNG)

**Comment by mbilker** [comment]
Hmm. I do not get these on Linux and I don't have my Windows 7 VM on hand to install N1 in it. What happens when you click on them?

**Comment by Gangwolf** [comment]
Ah, its seems to only appear when I have more than one account set up (I have one gmail, and another custom one). When I click on them they are unresponsive.

**Comment by mbilker** [comment]
![2015-11-17-195406_1280x800_scrot](https://cloud.githubusercontent.com/assets/1266541/11229599/fed3c64c-8d64-11e5-9437-74f9957360dd.png)
I see no additional menus under Linux. I have 2 accounts setup, Gmail and Google Apps domain.

**Comment by Gangwolf** [comment]
It just disappeared on me. Maybe toggling the menu bar? I tried restarting it but it's still gone, it seems to be random.

**Comment by mbilker** [comment]
Probably something with Electron. I am having weird issues writing a plugin.

**Comment by Gangwolf** [comment]
I think I found the cause, I restarted the app multiple times but it seemed go
away. I added another account and went into 'Preferences', and boom, there it
was, the menu items. Thus, I believe that adding an account causes menu items
to appear in program windows.  

> On Nov 17 2015, at 5:03 pm, mbilker &lt;notifications@github.com&gt; wrote:  

>

> Probably something with Electron. I am having weird issues writing a plugin.

>

> —  
Reply to this email directly or [view it on GitHub](https://github.com/nylas/N
1/issues/418#issuecomment-157563566).![](https://github.com/notifications/beac
on/AO9S8PLOMDk9butRdwdTpYZUEydpaqSsks5pG8XFgaJpZM4GkUue.gif)
